### DISCUSSION TITLE
Student repositories wanted for beta prototype

### DISCUSSION THREAD

**Discussion by ericdrosado** [community/community#17720] [GitHub Education]
GitHub Education wants to make it easier for students to get their start in the open source community. We are looking for student leaders who are interested in creating repositories to teach other students new skills or develop an open source project with other students. These repositories will live in GitHub, but they will also be exclusively showcased to the student community on GitHub Global Campus. 

This is an opportunity to:

* Showcase your work to the student community.
* Build your portfolio for internships and future employers.
* Grow your network.
* Become an open source maintainer.
* Help other students learn.

If you would like to submit a repository to be showcased to the student community or if you have an idea for a repository, [capture the details here]().

Students selected and contacted by GitHub Education will receive GitHub swag for participating.

**Comment by Guedesou** [comment]
# Loved this opportunity!
 ## I already worked as Community Manager in communities of young developers. 
## I already participate in Hackathons as Judge and avaliator, and this opportunity will help me to change others lives. Leave a legacy and help people its the best motivation in the world!(●'◡'●)
